test/commands/init.test.ts

```diff
@@ -0,0 +1,336 @@
+import { test } from '@oclif/test';
+import { expect } from 'chai';
+import { plainToClass } from 'class-transformer';
+import fs from 'fs';
+import yaml from 'js-yaml';
+import path from 'path';
+import sinon from 'sinon';
+import { ComponentConfigV1 } from '../../src/dependency-manager/src/component-config/v1';
+import { BuildSpecV1 } from '../../src/dependency-manager/src/service-config/v1';
+import { mockAuth } from '../utils/mocks';
+
+describe('init', function () {
+
+  // set to true while working on tests for easier debugging; otherwise oclif/test eats the stdout/stderr
+  const print = false;
+
+  let writeFileStub: sinon.SinonStub;
+
+  test
+    .do(ctx => mockAuth())
+    .finally(() => sinon.restore())
+    .stdout({ print })
+    .stderr({ print })
+    .command(['init', '--help'])
+    .it('succinctly describes the init command', ctx => {
+      expect(ctx.stdout).to.contain('Initialize an architect component from an existing docker-compose file\n')
+    });
```

Comment:
A test for help text contents might be overkill. More likely to get in the way then to be helpful.